CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ## Unreleased
 
 <!-- List New Changes Here -->
+* Build now delegates authority to the Analyzer for what urls are package external instead of local heuristics.
+* Bundling now processes files coming in from streams, to support things like js-minification before bundling.
 
```

Comment:
I don't see a specific bug for this.  It feels like we might need that behavior in both `polymer-build` AND `polymer-bundler`.  I'll add bugs to both.